README.md

```diff
@@ -112,9 +112,25 @@ org.freedesktop.tuhi1.Device
   Property: Listening (b)
       Indicates whether the daemon is currently listening for the device.
 
-      This property is set to True when a Listen() request initiates the
-      search for device connections. When the Listen() request completes
-      upon timeout, the property is set to False.
+      This property is set to True when a StartListening() request initiates
```

Comment:
What went with the discussion about dropping Listening in favor of the signal?

just asking...

Reply:
right now, only one client can listen, the rest gets `EAGAIN` on `StartListening`. `StopListening()` triggers the signal but it felt weird to have that broadcast to everyone. On success it makes sense but what's a client going to do with a signal that has an error code for a method it never invoked? Especially if a client isn't authorised to start, so everyone could see the failed attempts.

So right now, the approach is: check if `Listening` is `False`, then `StartListening()`. Otherwise, wait for the property to change.